src/EA.Weee.Web/Areas/Aatf/Views/Shared/_ContactProgressiveDisclosure_AddressOnly.cshtml

```diff
@@ -3,20 +3,21 @@
 }
 
 <div class="govuk-body govuk-!-font-size-16">
-    <b>England</b><br>
-    Environment Agency<br>
-    Producer Responsibility Regulatroy Services<br>
-    Telephone: 03708 506 506<br><br>
-    <b>Northern Ireland</b><br>
-    Northern Ireland Environment Agency<br>
-    Producer Responsibility Unit<br>
-    Telephone: 02 890 569338<br><br>
-    <b>Scotland</b><br>
-    Scottish Environment Protection Agency<br>
-    Producer Compliance and Waste Shipments Unit<br>
-    Telephone: 01786 457700<br><br>
-    <b>Wales</b><br>
-    Natural Resources Wales<br>
-    Producer Responsibility Unit<br>
+    Contact your environmental regulator<br /><br />
```

Comment:
@GuidoWorkDev Should the line "Contact your environmental regulator" have been added? Looks in Git that its a new addition.

Reply:
line removed